src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -394,7 +395,15 @@
         (re-frame/dispatch [::models.pin-message/show-pin-limit-modal chat-id]))
       (re-frame/dispatch [::models.pin-message/send-pin-message (assoc message :pinned (not pinned))]))))
 
-(defn on-long-press-fn [on-long-press {:keys [pinned message-pin-enabled outgoing edit-enabled show-input? community?] :as message} content]
+(defn on-long-press-fn [on-long-press {:keys [pinned
+                                              message-pin-enabled
+                                              outgoing
+                                              edit-enabled
+                                              show-input?
+                                              community?
+                                              community-id
+                                              can-manage-users?
+                                              from] :as message} content]
```

Comment:
[Code style] The Clojure Style Guide recommends breaking arguments over multiple lines for readability purposes. So something like this for the arguments' vector:

```clojure
[on-long-press
 {:keys [pinned
         message-pin-enabled
         outgoing
         edit-enabled
         show-input?
         community?
         community-id
         can-manage-users?
         from]
  :as   message}
 content]
```